packages/video_player/video_player/lib/video_player.dart

```diff
@@ -819,6 +926,63 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     }
   }
 
+  /// Gets the available audio tracks for the video.
+  ///
+  /// Returns a list of [VideoAudioTrack] objects containing metadata about
+  /// each available audio track. The list may be empty if no audio tracks
+  /// are available or if the video is not initialized.
+  ///
+  /// Throws an exception if the video player is disposed.
+  Future<List<VideoAudioTrack>> getAudioTracks() async {
+    if (_isDisposed) {
+      throw Exception('VideoPlayerController is disposed');
```

Comment:
should we be throwing an exception if the video is already disposed ? or should be just return like how its just returned in many of the other methods ?